payloadcms/payloadcms-dev/docker-compose.yml

```
@@ -0,0 +1,44 @@
+version: "3"
+
+services:
+  payload:
+    image: node:18-alpine
+    ports:
+      - "3000:${PORT:-3000}"
+    volumes:
+      - .:/home/node/app
+      - node_modules:/home/node/app/node_modules
+    working_dir: /home/node/app/
+    command: sh -c "yarn install && yarn dev"
+    depends_on:
+      - mongo
+    env_file:
+      - .env
+
+  mongo:
```

Can we name the image to know that it's the DEV one?

```suggestion
  mongo-payload-dev:
```